docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -238,17 +239,11 @@ When working with a [Vite](https://vitejs.dev) project you must make a few modif
     <Tabs.Item value="JavaScript">JavaScript</Tabs.Item>
   </Tabs.List>
   <Tabs.Panel value="TypeScript">
-    <Fragment>
-      {({ platform }) => import(`./shared/vite-polyfill.mdx`)}
-    </Fragment>
-    <Fragment>
-      {({ platform }) => import(`./shared/vite-polyfill-ts.mdx`)}
-    </Fragment>
+    <VitePolyfill />
```

Comment:
Should this be here?

Reply:
Yes the `<VitePolyfill />` component is currently being used for both `js` and `ts` and then the `<VitePolyfillTS />` component is additional information that is only relevant for `ts`.  This could be refactored so that only one or the other is used but instead I decided to just follow the pattern that is already in use.

Reply:
Can we add `VitePolyfill` inside `VitePolyfillTS`? Think the current approach might be confusing in the future since the names read as being two variants